### PR TITLE
update: authentication added

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Login and Sign up pages with validation added

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,132 +1,222 @@
...
+.nav-brand {
+    font-size: xx-large;
     font-weight: bold;
     font-family: var(--heading-font);
 }
-.nav-collapse{
-    padding:0 1rem;
+.nav-brand-link{
+    text-decoration: none;
+    color:var(--text-col);
```
Comment (sks-21) [inline style.css:40]: Seems you have created the navbar locally. If you have made a nav component in your CL then it's preferred to use that.

**Review by sks-21 — APPROVED** [review]
Rest everything looks nice. Good to merge.

**Review by ShreyashMahajan — APPROVED** [review]
code looks good to merge

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css:40]: No, I made a simple one and here I need to create another as shown in Figma sites

**Pull request merged**